networking/eth2/src/test/java/tech/pegasys/teku/networking/eth2/gossip/subnets/StableSubnetSubscriberTest.java

```diff
@@ -157,14 +112,15 @@ void shouldReplaceExpiredSubscriptionsWithNewOnes() {
   void shouldGenerateLargeNumberOfSubscriptionsAndCheckTheyreAllCorrect() {
```

Comment:
This one could be dropped, I'm not sure we could test here anything with the changes

Reply:
Done